test/functional/combine_logs.py

```diff
@@ -78,10 +77,11 @@ def read_logs(tmp_dir):
     for each of the input log files."""
 
     # Find out what the folder is called that holds the debug.log file
-    chain = glob.glob("{}/node0/*/debug.log".format(tmp_dir))
-    if chain:
-        chain = chain[0]  # pick the first one if more than one chain was found (should never happen)
-        chain = re.search('node0/(.+?)/debug\.log$', chain).group(1)  # extract the chain name
+    chain = ''
```

Comment:
Python doesn't have scope "blocks", so this is not needed. `chain` is in this functions' scope after assignment
```suggestion
```

Reply:
Thanks, removed the line.